index.md

```diff
@@ -22,7 +22,7 @@ Before joining Collaborative Lesson Development training, participants should be
 
 After attending this training, participants will be able to:
 
-- collaboratively develop and publish lessons using The Carpentries lesson infrastructure:
+- collaboratively develop and publish lessons using The Carpentries lesson infrastructure (aka [The Carpentries Workbench](https://github.com/carpentries/sandpaper-docs)):
```

Comment:
```suggestion
- collaboratively develop and publish lessons using The Carpentries lesson infrastructure (aka [The Carpentries Workbench](https://carpentries.github.io/workbench/)):
```

@anenadic please could you add this URL as a `[workbench]` entry to the `links.md` file and remake these links "reference style" using that target?

Reply:
Will do.